app/javascript/components/Jobs/JobLauncherItem.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { Component, Fragment } from 'react';
+import * as PropTypes from 'prop-types';
+
+class JobLauncherItem extends Component {
+    render() {
+        const j = this.props.job; // Save some characters
+
+        var confirmationDialogProps = {}
+        if (j.confirmation_dialog) {
+            confirmationDialogProps = {
+                "data-confirm": j.confirmation_dialog
+            };
+        }
+        return (
+            <tr>
+                <td><span className="job_name" data-toggle="tooltip" title={`${j.job_description}`} >{j.job_name}</span></td>
+                <td>{j.last_run}</td>
+                <td>
+                    <a
+                        {...confirmationDialogProps}
+                        rel="nofollow"
+                        data-method="post"
+                        href={`${this.props.run_url_prefix}?job_name=${j.job_short_name}`}>
```

Comment:
This is kinda hacky. It would probably be better to use an `onClick` handler here and make that call as an xhr request within it so that when the request completes you can do something like update the job log etc like this:
```javascript
onJobRun = () => {
    FooService.runJob(bar).then(() => {
        this.props.updateJobLog(baz);    
    }
}
```

Reply:
Fair enough... would you be ok if I add that as a separate issue and do it as a refactor later?    We are kind of on a roll here with getting instrumentation into the app that we need to write a paper.   I acknowledge that this way is better, and I think we should do it--if I understand correctly, this would also mean that you don't have to refresh the page in order to see the new job added to the job log, though you would have to still (for now) refresh the page in order to see the job progress.  
We might be able to fix that as well in some future issue.

I'll add both of those issues a link to them in a comment.

Reply:
Ok, please see issues #281 and #282  ... is that enough to withdraw the need to do that in this PR?

Reply:
Yeah, it works for now.